templates/individual_response/how.html

```diff
@@ -0,0 +1,67 @@
+{% extends 'layouts/_base.html' %}
```

Comment:
Did you try using the `question.html` template? (it provides the error message handling and block rendering)